frontend/src/user/use-user.ts

```diff
@@ -1,36 +1,49 @@
 import {useOnce} from "../hooks/use-once";
-import {useState} from "react";
+import {useState, useEffect} from "react";
+import {User, userFromJson } from "zero-zummon"
 
 type UseUserReturn = {
     isLoading: boolean,
     isLoggedIn?: boolean,
     username?: string,
+    isAdmin?: boolean,
 }
 
 export const useUser = (): UseUserReturn => {
+    const [userId, setUserId] = useState<null>()
+
     const [state, setState] = useState<UseUserReturn>({
         isLoading: true,
         isLoggedIn: undefined,
         username: undefined,
+        isAdmin: false,
     })
 
     useOnce(async () => {
         try {
             const response = await fetch(import.meta.env.VITE_ZTOR_URL + "/user-info", {
```

Comment:
When I log in with a Keycloak user who is not in the database, /user-info gives a 500 error with `java.util.NoSuchElementException`.

Probably it should just have `isAdmin: false`

Reply:
Please fix this in the backend not in the frontend